config/source/cli/plugins.go

```diff
@@ -0,0 +1,32 @@
+// Package cli provides cli.
+package cli
+
+import (
+	"errors"
+
+	"go-micro.dev/v5/log"
+	"go-micro.dev/v5/util/container"
+)
+
+var (
+	// ErrFlagExists is returned when the given element exists in the flag container.
+	ErrFlagExists = errors.New("element exists already")
+
+	// Plugins contains source/cli subplugins, for example urfave, pflag, cobra.
+	Plugins = container.NewMap[ParseFunc]() //nolint:gochecknoglobals
+	// Flags is the global flag container where you have to register flags with.
+	Flags = container.NewList[*Flag]() //nolint:gochecknoglobals
+)
```

Comment:
You can put the `nolint` directive above the `var(`, it will apply to the entire block.
Also either use newlines for every global or for none.

Reply:
Actually I would put errors and other global in separate blocks

Reply:
Explicit is better than implicit on nolint lines I believe. You'r right about the newline.